utility/src/main/java/org/oppia/android/util/logging/BUILD.bazel

```diff
@@ -0,0 +1,97 @@
+"""
+General purpose utilities for logging.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "annotations",
+    srcs = [
+        "LoggingAnnotations.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//third_party:javax_inject_javax_inject",
+    ],
+)
+
+kt_android_library(
+    name = "console_logger",
+    srcs = [
+        "ConsoleLogger.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        ":annotations",
+        ":log_level",
+        "//third_party:javax_inject_javax_inject",
+        "//third_party:org_jetbrains_kotlinx_kotlinx-coroutines-core",
+        "//utility/src/main/java/org/oppia/android/util/threading:annotations",
+    ],
+)
+
+kt_android_library(
+    name = "event_bundle_creator",
+    srcs = [
+        "EventBundleCreator.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//model:event_logger_java_proto_lite",
+    ],
+)
+
+kt_android_library(
+    name = "event_logger",
+    srcs = [
+        "EventLogger.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//model:event_logger_java_proto_lite",
+    ],
+)
+
+kt_android_library(
+    name = "exception_logger",
+    srcs = [
+        "ExceptionLogger.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+)
+
+kt_android_library(
+    name = "log_level",
+    srcs = [
+        "LogLevel.kt",
+    ],
+)
+
+kt_android_library(
+    name = "logger_module",
```

Comment:
Suggest naming this "prod_module" instead, for parity with other modules that we expose for providing production implementations.

Reply:
Done